src/main/kotlin/uk/gov/gdx/datashare/controllers/AdminController.kt

```diff
@@ -11,80 +10,17 @@ import org.springframework.http.MediaType
 import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.web.bind.annotation.*
 import uk.gov.gdx.datashare.config.ErrorResponse
-import uk.gov.gdx.datashare.models.CognitoClientRequest
 import uk.gov.gdx.datashare.models.CreateAcquirerRequest
 import uk.gov.gdx.datashare.models.CreateSupplierRequest
-import uk.gov.gdx.datashare.repositories.AcquirerEvent
-import uk.gov.gdx.datashare.repositories.AcquirerEventRepository
 import uk.gov.gdx.datashare.services.AdminService
 
 @RestController
 @RequestMapping("/admin", produces = [MediaType.APPLICATION_JSON_VALUE])
 @PreAuthorize("hasAnyAuthority('SCOPE_events/admin')")
 @Tag(name = "13. Admin")
 class AdminController(
-  private val acquirerEventRepository: AcquirerEventRepository,
   private val adminService: AdminService,
 ) : BaseApiController() {
-  @GetMapping("/events")
-  @Operation(
-    summary = "Get Events",
-    description = "Need scope of events/admin",
-    responses = [
-      ApiResponse(
-        responseCode = "200",
-        description = "Event",
-        content = [
-          Content(
-            mediaType = "application/json",
-            array = ArraySchema(schema = Schema(implementation = AcquirerEvent::class)),
-          ),
-        ],
-      ),
-      ApiResponse(
-        responseCode = "415",
-        description = "Not able to process the request because the payload is in a format not supported by this endpoint.",
-        content = [
-          Content(
-            mediaType = "application/json",
-            schema = Schema(implementation = ErrorResponse::class),
-          ),
-        ],
-      ),
-    ],
-  )
-  fun getEvents() = acquirerEventRepository.findAll().toList()
-
-  @PostMapping("/cognitoClients")
-  @Operation(
-    summary = "Create cognito client",
-    description = "Need scope of events/admin",
-    responses = [
-      ApiResponse(
-        responseCode = "200",
-        description = "Cognito client details",
-      ),
-      ApiResponse(
-        responseCode = "415",
-        description = "Not able to process the request because the payload is in a format not supported by this endpoint.",
-        content = [
-          Content(
-            mediaType = "application/json",
-            schema = Schema(implementation = ErrorResponse::class),
-          ),
-        ],
-      ),
-    ],
-  )
-  fun createCognitoClient(
-    @Schema(
-      required = true,
-      implementation = CognitoClientRequest::class,
-    )
-    @RequestBody
-    cognitoClientRequest: CognitoClientRequest,
-  ) = adminService.createCognitoClient(cognitoClientRequest)
```

Comment:
I assume we're still doing this as part of the "onboard an acquirer" workflows? If not, can we reduce some roles/generally tighten access?

Reply:
Yeah, still doing it there, but we currently have no use case for doing it separately (this won't rotate creds)